docs/diagrams/markAttendanceActivityDiagram.puml

```diff
@@ -0,0 +1,22 @@
+@startuml
+'https://plantuml.com/activity-diagram-beta
+
+start
+:User inputs attendance;
+:npc_track parses attendance by index, tutorial and status;
+if (Is the tutorial valid?) then (Tutorial is valid)
+    if(Tutorial status is valid?) then (Tutorial status is invalid)
```

Comment:
the content inside if is supposed to be blank according to what is taught.

the content inside then is supposed to be in a [ ] according to what is taught.

Reply:
Do you think we need to add in a condition for isIndexValid too?